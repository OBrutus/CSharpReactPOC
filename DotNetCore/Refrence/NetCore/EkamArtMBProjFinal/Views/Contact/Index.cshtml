<div class="col-md-6">
    <div class="container">
        <h2>Get in Touch!</h2>
        <ul>
            <li>
                <i class="fa fa-home" aria-hidden="true"></i>
                Ekam Art Studios,
                Bangalore,
                KA - 560078
            </li>
            <li>Get in touch with us <a>+91 90 354567</a> </li>
            <li><a href="mailto://ezhil@gmail.com">ezhil@gmail.com</a></li>
            <email address="ezhil@gmail.com" content="Mail us"></email>
           
        </ul>
    </div>
</div>