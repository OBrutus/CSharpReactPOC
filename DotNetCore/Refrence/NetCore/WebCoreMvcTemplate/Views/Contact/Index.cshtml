<div class="col-md-6">
    <div class="container">
        <ul>
            <li>
                Emovies Studio,
                Bangalore,
                India
            </li>
            <li>Get in touch with us <a>+91 90 354567</a> </li>
            <li><a href="mailto://ezhil@gmail.com">ezhil@gmail.com</a></li>
           <email address="ezhil@gmail.com" content="Mail us"></email>
        </ul>
    </div>
</div>