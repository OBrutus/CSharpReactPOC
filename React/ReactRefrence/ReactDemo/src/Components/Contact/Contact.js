import React, { Fragment } from "react";

//Create component - class

class Contact extends React.Component {
  render() {
    return (
      // React Fragment
      <>
        <h2> Contact us for any Queries!</h2>
        <h3>Call us at 123345</h3>
      </>
    );
  }
}

export default Contact;
