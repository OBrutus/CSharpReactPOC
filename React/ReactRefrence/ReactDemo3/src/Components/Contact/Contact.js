import React, { Fragment } from "react";

//Create component - class

class Contact extends React.Component {
  render() {
    return (
      <div className="jumbotron">
        <h2>Contact us for any queries</h2>
        <h3>Call:123456</h3>
      </div>
    );
  }
}

export default Contact;
