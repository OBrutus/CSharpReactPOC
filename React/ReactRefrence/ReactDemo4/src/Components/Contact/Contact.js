import React, { Fragment } from "react";

//Create component - class

class Contact extends React.Component {
  render() {
    return (
      <div className="jumbotron">
        <h2>Contact us for any queries</h2>
        <h3>Call:123456</h3>
        <h3>Mail us at react@support.com
          
        </h3>
      </div>
    );
  }
}

export default Contact;
