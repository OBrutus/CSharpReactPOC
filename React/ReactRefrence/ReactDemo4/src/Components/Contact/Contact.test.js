import React from "react";
import renderer from "react-test-renderer";
import Contact from "./Contact";

test("Creating a snapshot of Contact Component", () => {
  const component = renderer.create(<Contact />);
  const treesnapshot = component.toJSON();
  expect(treesnapshot).toMatchSnapshot();
});
