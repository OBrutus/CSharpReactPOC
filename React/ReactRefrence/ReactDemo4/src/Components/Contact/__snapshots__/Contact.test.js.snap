// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creating a snapshot of Contact Component 1`] = `
<div
  className="jumbotron"
>
  <h2>
    Contact us for any queries
  </h2>
  <h3>
    Call:123456
  </h3>
  <h3>
    Mail us at react@support.com
  </h3>
</div>
`;
